### ISSUE TITLE
Benchmark Models for LSTM, GPT, S4

### ISSUE THREAD

**Issue by KSUN63** [molML/s4-for-de-novo-drug-design#9]
Hi there,

I really enjoyed reading your work and great job for you all to put forth S4 for molecular generation. As for the table 1 in your table, do you have any pretrained model, or can you point me to some pretrained code space to reproduce those results? I'm interested in the effect of softmax temperature on generation and it would be fun to play around with all three models shown in table 1. Thanks in advance!

Cheers,
Oliver

**Comment by rizaozcelik** [comment]
Dear Oliver,

I am attaching a zip file that contains:

- pretrained model weights (tf.keras)
- scripts needed to load and design with the models. The temperature parameter is already defined, so you can easily experiment with it.

Please let me know if you need anything further, and close the issue otherwise, so that I know I did help you 😄 

Have fun!

**Comment by KSUN63** [comment]
Dear Riza,

Thank you so much! I don't really see the zip file here, though. Is there something that I am missing here? If there's a file limit I am happy to leave my emails or whatever way that's easy. Thanks again!

Best,
Oliver

**Comment by KSUN63** [comment]
My email is kysun@berkeley.edu just in case if this is easier.

**Comment by rizaozcelik** [comment]
Dear Oliver,

Sorry for the inconvenience. File upload (apparently) didn't work here due to the file size limit. I just sent it as an email.

Closing this issue now 🙂
